src/System/i18n/config.ts

```diff
@@ -0,0 +1,24 @@
+import { getENV } from "Utils/getENV"
+
+// TODO: Serve static assets on the server-side
+const translations = require("System/i18n/locales/en-US/translation.json")
+
+export default {
+  debug: getENV("ENABLE_I18N_DEBUG") === "true",
+  resources: {
+    ["en-US"]: {
+      translation: translations,
+    },
+  },
+  detection: {
+    order: ["querystring"],
+    lookupQuerystring: "locale",
+  },
```

Comment:
We won't be using this option? Is there anything else in this config can we remove?